Image Viewing GUI/main.py

```diff
@@ -0,0 +1,88 @@
+# importing the modules
+
+from tkinter import *
+from PIL import ImageTk, Image
+
+root = Tk()
+root.title('Image viewer')
+root.resizable(0, 0)
+
+# opening the images from the image folder
+
+img1 = Image.open("images/img1.jpg")
```

Comment:
The image path needs to be fixed. Do this for all:
```suggestion
img1 = Image.open("./Image Viewing GUI/images/img1.jpg")
```